docs/contracts.rst

```diff
@@ -1,8 +1,72 @@
+.. _contracts:
+
 Contracts
 =========
 
 .. py:module:: web3.contract
 
+It is worth taking your time to understand all about contracts. To get started,
+check out this example:
+
+.. _contract_example:
+
+Simple Contract Example
+-----------------------
+
+.. code-block:: python
+
+    import json
+    import web3
+
+    from web3 import Web3, TestRPCProvider
+    from solc import compile_source
```

Comment:
Most beginners just install `py-solc` without installing the solidity complier. Should there be a note pointing to [Installing the Solidity Compiler](solidity.readthedocs.io/en/latest/installing-solidity.html)?